### DISCUSSION TITLE
Fish autocomplete

### DISCUSSION THREAD

**Discussion by ZeroDeltaAlpha** [martindstone/pagerduty-cli#45] [Ideas]
Hi,

I see that oclif can do fish completions? Is this something that could be implemented ?

**Comment by martindstone** [comment]
Hi @ZeroDeltaAlpha I suspect maybe it comes with the newest version of the framework? I will take a look, I know I was able to get bash and zsh completions so it's probably the same plugin...

**Comment by ZeroDeltaAlpha** [comment]
Thanks @martindstone for looking into this, this tool has been a lifesaver interacting with PagerDuty

**Comment by martindstone** [comment]
Hi @ZeroDeltaAlpha I am looking around and I am seeing a bunch of PR's for oclif to include fish autocomplete support but I don't see any merged, and I see that the current version of the oclif autocomplete plugin still says it only supports bash and zsh... can you point me to anything you've read on the topic?

**Comment by ZeroDeltaAlpha** [comment]
Hi @martindstone, i did see a third party library (https://github.com/MunifTanjim/oclif-plugin-completion) that allowed generation of fish completions but was last committed to 3 years ago, i think that is where i seen fish mentioned initially. The only relevant PR i can see about fish is https://github.com/oclif/plugin-autocomplete/pull/175, I'm hopeful this might get merged as the last conversation was September this year.
